cmd/common.go

```diff
@@ -322,3 +325,97 @@ func GetPastelInfo(ctx context.Context, config *configs.Config) (structure.RPCGe
 	}
 	return getifno, nil
 }
+
+// CheckPastelDRunning whether pasteld is running
+func CheckPastelDRunning(ctx context.Context, config *configs.Config) (ret bool) {
+	var failCnt = 0
+	var err error
+
+	log.WithContext(ctx).Info("Waiting the pasteld to be started...")
+
+	for {
+		if _, err = RunPastelCLI(ctx, config, "getinfo"); err != nil {
+			time.Sleep(10000 * time.Millisecond)
```

Comment:
could you simply change  `10000 * time.Millisecond` to `10*time.Second`?

Reply:
ok, will change - that was in code that way already :)